src/Language/PureScript/Kinds.hs

```diff
@@ -18,8 +20,8 @@ data Kind
   | Row Kind
   -- | Function kinds
   | FunKind Kind Kind
-  -- | Type-level strings
-  | Symbol
+  -- | A named kind
+  | NamedKind (Qualified (ProperName 'KindName))
```

Comment:
In updating this I've found that I'd missed updating `primImports` in `Sugar/Names/Env.hs`.